Description: fix formatting error and typos in the manpages
Author: Julien Valroff <julien@debian.org>
        Giovani Augusto Ferreira <giovani@riseup.net>
        Samuel Henrique <samueloph@debian.org>
        Fukui Daichi <a.dog.will.talk@akane.waseda.jp>
Last-Updated: 2022-8-31
diff --git a/man/fr/unhide.8 b/man/fr/unhide.8
index 5c06ffa..483c4a2 100644
--- a/man/fr/unhide.8
+++ b/man/fr/unhide.8
@@ -222,7 +222,7 @@ Test standard :
 unhide sys proc
 .TP
 Test le plus complet :
-unhide -m -d sys procall brute reverse
+unhide \-m \-d sys procall brute reverse
 .SH "BUGS"
 .PP
 Rapportez les bugs de \fBunhide\fR sur le bug tracker de GitHub (https://github.com/YJesus/Unhide/issues)
diff --git a/man/unhide-tcp.8 b/man/unhide-tcp.8
index 46ae799..05be2e5 100644
--- a/man/unhide-tcp.8
+++ b/man/unhide-tcp.8
@@ -18,35 +18,35 @@ given on the command line.
 .PP
 .SH "OPTIONS"
 .TP
-\fB\-h --help\fR
+\fB\-h -\-help\fR
 Display help
 .TP
-\fB\--brief\fR
+\fB\-\-brief\fR
 Don't display warning messages, that's the default behavior.
 .TP
-\fB\-f --fuser\fR
+\fB\-f -\-fuser\fR
 Display fuser output (if available) for the hidden port
 On FreeBSD, instead of fuser command, displays the output of the sockstat command for the hidden port.
 .TP
-\fB\-l --lsof\fR
+\fB\-l -\-lsof\fR
 Display lsof output (if available) for the hidden port
 .TP
-\fB\-n --netstat\fR
+\fB\-n -\-netstat\fR
 Use /bin/netstat instead of /sbin/ss. On system with many opened ports, this can
 slow down the test dramatically.
 .TP
-\fB\-s --server\fR
+\fB\-s -\-server\fR
 Use a very quick strategy of scanning. On system with a lot of opened ports,
 it is hundreds times faster than ss method and ten thousands times faster than
 netstat method.
 .TP
-\fB\-o --log\fR
+\fB\-o -\-log\fR
 Write a log file (unhide-tcp-AAAA-MM-DD.log) in the current directory.
 .TP
-\fB\-V --version\fR
+\fB\-V -\-version\fR
 Show version and exit
 .TP
-\fB\-v --verbose\fR
+\fB\-v -\-verbose\fR
 Be verbose, display warning message (default : don't display).
 This option may be repeated more than once.
 .PP
diff --git a/man/unhide.8 b/man/unhide.8
index c4d8c1f..e263de5 100644
--- a/man/unhide.8
+++ b/man/unhide.8
@@ -29,7 +29,7 @@ Display help
 Do more checks. As of 2012\-03\-17 version, this option has only
 effect for the procfs, procall, checkopendir and checkchdir tests.
 .br
-Implies -v
+Implies \-v
 .TP
 \fB\-r\fR
 Use alternate version of sysinfo check in standard tests
@@ -224,7 +224,7 @@ Standard test:
 unhide sys proc
 .TP
 Deeper test:
-unhide -m -d sys procall brute reverse
+unhide \-m \-d sys procall brute reverse
 .SH "BUGS"
 .PP
 Report \fBunhide\fR bugs on the bug tracker on GitHub (https://github.com/YJesus/Unhide/issues)
